### ISSUE TITLE
Files scattered, not following the format

### ISSUE THREAD

**Issue by Silver077** [swapnanildutta/Hackerrank-Codes#306]
The Java section's files are scattered, some are even duplicated. 
I already made a PR to fix this, but like for real yesterday, I fixed some next day there's more. They didn't even put up the HR link in README.md
I hope the mods can double-check before approving PRs like that

**Comment by swapnanildutta** [comment]
@Silver077 Could you please point out the folders where there is this issue.

**Comment by swapnanildutta** [comment]
@Silver077 Thank you for arranging the files, it would be great if you continue to do so. That will be of great help.

**Comment by Silver077** [comment]
No problem, I will continue to help out this repository (this helped me a lot).

I sorted some more files, didn't even notice how many duplicated lines there were last time.
https://github.com/swapnanildutta/Hackerrank-Codes/pull/338

Also, I am having a hard time passing the codacy auto-check due to:
[list-item-indent] Incorrect list-item indent: add 2 spaces
I followed the pattern and I still get the errors, I usually resolve it by adding a blank and delete it and doing it multiple times,
but I still can't figure out what is wrong.
